specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json

```diff
@@ -2393,14 +2707,91 @@
         },
         "nodes": {
           "description": "total nodes purchased",
-          "type": "integer"
+          "type": "integer",
+          "readOnly": true
```

Comment:
I assume this is a client side bugfix and not a (breaking) change of a property to read-only?

Reply:
this property was never modifiable from client side. So this is not breaking change